cypress/support/cypress-commands/commands.js

```diff
@@ -699,7 +699,11 @@ Cypress.Commands.add('launchApp', (appType, appCallSign) => {
         },
       },
     };
-    requestMap.params.intent.data = data;
+    const messageIntent = {
+      action: CONSTANTS.SEARCH,
+      data: data,
+    };
```

Comment:
do we really need to create messageIntent twice ? https://github.com/rdkcentral/firebolt-certification-suite/pull/138/files#diff-a0c4b6497a6be8ded74b56b3268804337f0607a41306ae7c236b91b7a7fab118R678
Cant we just create the data object before and then at the last create a single messageIntent

Reply:
Addressed above review comment